packages/settings-library/src/settings_library/r_clone.py

```diff
@@ -8,8 +8,8 @@
 
 class S3Provider(str, Enum):
     AWS = "AWS"
-    CEPH = "CEPH"
-    MINIO = "MINIO"
+    CEPH = "Ceph"
```

Comment:
How these strings change affect/fix this sisue?  These are just identifiers, right? AFAIK could eventually be even ints

The actual information on the provider is (correctly) setup apart e.g. with this map
https://github.com/ITISFoundation/osparc-simcore/blob/cf422bb2b61b8ae738eaad79f34f1de7c2bce995/packages/settings-library/src/settings_library/utils_r_clone.py#L17

Reply:
Very good point will refactor to use these!